_includes/campaign-form.html

```diff
@@ -54,17 +54,21 @@
 
     <label v-if="campaign==='mp'" for="service-name">[{$t("contempt_notice_msg")}] : <span class="service-name">[{personName}], [{$t("constituencies." + constituencyCode)}]</span></label>
 
-    <button id="btn-email-mobile" class="btn-email btn-primary" v-if="mobile"
-      v-on:click.stop.prevent="sendEmail()">[{$t("send_email")}]</button>
+    <a id="btn-email-mobile" class="btn btn-email btn-primary" v-if="mobile"
+      v-bind:href="fullmailtourl"
```

Comment:
No _blank on mobile.

Reply:
On mobiles we are using `mailto:` anyway, so it doesn't matter

Reply:
Yup, didn't want to open a new tab unnecessarily.